### ISSUE TITLE
a11y: Textfields show up as multiple semantics nodes

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#14211]
Should be just one.

Probably caused by the Scrollable that is surrounding a text field.

Problem is more noticeable on iOS where the different nodes can be focused separately. Android doesn't recognize some of the extra nodes as focusable. 

/cc @yjbanov
